docs/25_git_lfs.md

```diff
@@ -0,0 +1,107 @@
+
+### Background story
+
+ACME Incorporated is planning on using Toggl for time tracking and are using this repository to provide information on how to use Toggl and download the desktop application that has passed the IT Security team's audit.
+
+Unfortunately, the `.dmg` file was added to the Git repository without using Git LFS and we need to fix that mistake. Perform the steps found in the upcoming sections to successfully retain the `.dmg` file in the project but utilizing Git LFS to maintain it in the project.
+
+### Activity: Install Git LFS
+
+Before we can begin using Git LFS, we need to install it on our local machine. Below are the steps for installing Git LFS on your machine based on your operating system.
+
+#### MacOS
+
+1. Navigate to [git-lfs.github.com](https://git-lfs.github.com/) and click **Download**. Alternatively, you can install Git LFS using a package manager:
+
+    - To use [Homebrew](Homebrew), run `brew install git-lfs`.
+    - To use [MacPorts](https://www.macports.org/), run `port install git-lfs`.
+
+    > If you install Git LFS with Homebrew or MacPorts, skip to step six.
+
+1. On your computer, locate and unzip the downloaded file.
+1. Open Terminal.
+1. Change the current working directory into the folder you downloaded and unzipped.
+
+    ```sh
+    cd ~/Downloads/git-lfs-1.X.X
+    ```
+
+    > Note: The file path you use after `cd` depends on your operating system, Git LFS version you downloaded, and where you saved the Git LFS download.
+
+1. To install the file, run this command:
+
+    ```sh
+    $ ./install.sh
+    > Git LFS initialized.
+    ```
+
+    > Note: You may have to use `sudo ./install.sh` to install the file.
+
+1. Verify that the installation was successful:
+
+    ```sh
+    $ git lfs install
+    > Git LFS initialized.
+    ```
+
+#### Windows
+
+1. Navigate to [git-lfs.github.com](https://git-lfs.github.com/) and click **Download**.
+
+   > Tip: For more information about alternative ways to install Git LFS for Windows, see this [Getting started guide](https://github.com/github/git-lfs#getting-started).
+
+1. On your computer, locate the downloaded file.
+1. Double click on the file called _git-lfs-windows-1.X.X.exe_, where 1.X.X is replaced with the Git LFS version you downloaded. When you open this file Windows will run a setup wizard to install Git LFS.
+1. Open Git Bash.
+1. Verify that the installation was successful:
+
+```sh
+$ git lfs install
+> Git LFS initialized.
+```
+
+## Tracking files with Git LFS
+
+Before we tell Git LFS what files to track, we should identify what files are taking up the most space in our repository.
+
+### Activity: Exploring the repository files
+
+1. Open your command line application.
+1. Change your current working directory to the `lfs-example` repository.
+1. Run `git lfs migrate info`, your output should look similar to:
+
+    ```sh
+    migrate: Fetching remote refs: ..., done.                               migrate: Sorting commits: ..., done.
```

Comment:
ugh why does this line look weird, it looks fine in my vscode after I fix the tab issues for the linter

Reply:
think i fixed it